.travis.yml

```diff
@@ -87,6 +91,18 @@ jobs:
         BACKEND=sqlite
         PYTHON_VERSION=3.6
       stage: test
+    - name: "Tests [Sqlite][3.7][multiprocessing(spawn)]"
+      env: >-
+        AIRFLOW__CORE__MP_START_METHOD=spawn
```

Comment:
It will not work the way you think. Those environment variables are set in the Host not inside docker - you would have to also change docker-compose/base.yaml in order to propagate those variables to docker. However I think you should not do it this way. It should be enough to run ONE dag test with spawn method, not necessarily all of them. Running all the tests will take a lot of extra time needed on CI.

What I think should work is simply one "dag" test where os.environment is overridden for the time of execution. Internally those tests run separate process with "airflow run" so I think the env variable should be propagated to the new process. And one test is enough.

Do you need some more pointers/help with that? 

I am happy to help more if needed.

Reply:
Do you mean to run one test outside docker container?
I think `tests/jobs/test_scheduler_job.py` and `tests/utils/test_dag_processing.py` are better to run.
I saw some varieties of spawn-related issue with these test-file.

Reply:
No no - I mean to simply copy an existing test that runs one of test dags but to add this line:
```
os.environment['AIRFLOW__CORE__MP_START_METHOD]='spawn'
```

Reply:
Ah, I understand. Thanks.

Reply:
@potiuk I tried to do what you suggested but I noticed It's not enough because the pickle error happens when scheduler runs.
Instead, I added one test case to `tests/jobs/test_scheduler_job.py`, which is almost a copy of an existing test case but set `mp_start_method` to `spawn`.
The pickle error can be tested the test case but another issue, updated configuration is not inherited by child processes, cannot be tested so I also added a test case to `tests/utils/test_dag_processing.py`.

Reply:
Fantastic! I wait for last comments of @ashb but for me it's good to go!